src/System.Private.CoreLib/shared/System/Globalization/CultureData.Windows.cs

```diff
@@ -308,7 +308,7 @@ private string GetLanguageDisplayName(string cultureName)
                 ((ci = GetUserDefaultCulture()) != null) &&
                 !CultureInfo.DefaultThreadCurrentUICulture.Name.Equals(ci.Name))
             {
-                return SNATIVEDISPLAYNAME;
+                return NativeName;
```

Comment:
Same for these; they should maintain the same casing/spelling as the Windows names, as that's what they represent.